### PR TITLE
theamericanshortstory06 initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
The latest installment in The American Short Story series.

### CODE REVIEW

File: notes/stories/television/tv-theamericanshortstory.st.txt [hunk 1]
```diff
@@ -229,3 +229,47 @@ sports [A group of friends were shown enjoying a leisurely game of croquet.]
...
+
+:: Ratings
+4 <brent>
+4 <paul>
+
+:: Major Themes
+growing as a person [The story culminates with Andy realizing he was a fool for having tried to woo his love interest, Lucy, with lies of importance and wealth.]
+romantic love [Andy was smitten by Lucy, and Lucy was head over heels for the man who Andy feigned to be. Andy waxed poetic about how he wanted his first time to be with a princess.]
+mentor and protégé [Burt taught Andy the ins and outs of training race horses.]
+rite of passage into manhood [Andy went from being a young, bashful, and sexually inexperienced horse trainer to being a cigar-smoking, whiskey-drinking son of a gun, a man of the world and about town if you will, who was not above wooing girls at the horse races with lies of importance and wealth.]
```
Comment (odinlake) [inline notes/stories/television/tv-theamericanshortstory.st.txt:257]: The comment sounds like https://www.themeontology.org/theme.php?name=coming%20of%20age but it is unclear what the "rite of passage" was

Comment (odinlake) [reply notes/stories/television/tv-theamericanshortstory.st.txt:257]: examples of rites we've noticed: Vulcan warrior rite, killing a man, putting on "manly robes".

File: notes/stories/television/tv-theamericanshortstory.st.txt [hunk 2]
```diff
@@ -229,3 +229,47 @@ sports [A group of friends were shown enjoying a leisurely game of croquet.]
...
+
+:: Minor Themes
+racism [Andy's family was ashamed of him working with a black man.]
+gender issue [Andy explained that he wanted to be the strong one in a relationship and for his future wife to be timid. Andy first introduced himself to Lucy's brother rather than speak to her directly.]
+music [Burt was fond of singing while they traveled from town to town.]
+fiancée and fiancée [Henry and Eleanor were an agreeable couple who seemed pleased that Lucy had a suitor.]
+friendship [Andy encouraged a drunken Burt to pursue his dream of being a jockey after practice one day. Burt gave Andy a great deal of useful life advice in a sometimes gruff, sometimes jolly manner.]
+brother and sister [Lucy West accompanied Henry and Eleanor to the race track.]
+gambling [Andy and his new friends bet a lot of money on Andy's horse and won big.]
+lying [Andy told a whopper of a lie to woo his love interest, and ultimately came to regret it.]
```
Comment (odinlake) [inline notes/stories/television/tv-theamericanshortstory.st.txt:267]: "growing as a person" comment makes me wonder if "lying" shouldn't be major..?

**Review by odinlake — COMMENTED** [review]
comments inline

File: notes/stories/television/tv-theamericanshortstory.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-theamericanshortstory.st.txt:257]: Yes, in retrospect, I think it is reasonable to use **coming of age** as a major theme with the present justification for **rite of passage** as motivation, and move **rite of passage** to minor themes with a motivation to be written.

File: notes/stories/television/tv-theamericanshortstory.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-theamericanshortstory.st.txt]: Not an unreasonable suggestion as it was much featured in the last third or so of the story and was also relevant to to conclusion. I knocked it up.

File: notes/stories/television/tv-theamericanshortstory.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/television/tv-theamericanshortstory.st.txt]: I ended up deleting **rite of passage into manhood** on account of it being too tricky to motivate. I'll leave the pleasure of merging to you.

**Review by odinlake — APPROVED** [review]

**Pull request merged**